local-setup/nginx/sites-enabled/aether.conf

```diff
@@ -12,7 +12,7 @@
 server {
   listen                    80;
   charset                   utf-8;
-  server_name               aether.local;
+  server_name               aether.local 192.168.178.118;
```

Comment:
Taking a look at nginx docs, maybe it's possible to include the wildcard in the IP address too. :thinking: 

https://nginx.org/en/docs/http/server_names.html

Reply:
Do you want to keep this hardcoded IP here or try to find the working regexp, wildcard?

Reply:
Oh, actually I didn't even mean to leave that there... It's in the README, so that should be enough